pkg/db/migrations/migration_structs.go

```diff
@@ -38,6 +43,23 @@ var MigrationList = []*gormigrate.Migration{
 	alterEventInstances(),
 }
 
+// CleanUpDirtyData clean up the dirty data before migrating the tables.
+// When we add new constraints to old tables, we should especially consider the possibility of dirty data.
+func CleanUpDirtyData(db *gorm.DB) error {
```

Comment:
that can be risky to cleanup the data. Let us have a talk tomorrow.

Reply:
leave this interface to handle situations where dirty data may appear in the future.